engine/combat/appraisals/valere/moonerang.py

```diff
@@ -17,7 +17,7 @@
 
 
 class Moonerang(SoSAppraisal):
-    HIT_AT_POSITION_VALUE = 0.82
+    HIT_AT_POSITION_VALUE = 0.50
```

Comment:
What fight, if any, is this optimized for?

Reply:
It's sort of a lowest common denominator, so far i have encountered only one enemy position where it misses the first reflect and cannot start the chain but for all the other ones it seems to result in a longer (or equally long) chain.
I expect we'll be raising this again once we encounter larger distances but maybe we'll already have a more stable solution by then.